### PR TITLE
Add COMPATIBLE_CHAIN_ID into web3.env

### PR DESCRIPTION
**Pull request by Flouse**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -134,7 +134,9 @@ start: workspace/bin/godwoken
 	source ./gw_util.sh && start
 	make show_wait_tips
 # TODO: only deposit for dev accounts one time
-	source ./gw_util.sh && deposit_for_two_dev_accounts
+	if [ "$(GITHUB_RUN_ID)" != "" ] ; then \
+		source ./gw_util.sh && deposit_for_two_dev_accounts ; \
+	fi
```
Comment (keroro520) [inline Makefile:139]: godwoken-test workflow already deposits for test accounts: https://github.com/nervosnetwork/godwoken-tests/blob/develop/.github/workflows/reusable-integration-test-v1.yml#L197-L199

This line can be removed.

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Flouse) [reply Makefile]: done

**Review by keroro520 — APPROVED** [review]

File: docker/.build.mode.env [hunk 2]
```diff
@@ -62,4 +62,4 @@ ENABLE_GW_READONLY_NODE=false
 #
 # - WATCH_CKB_REORG [boolean] 
 #	if set to true, will start watch ckb network for reorgs 
-#	and output the logs in chain-reorg.log 
+#	and output the logs in chain-reorg.log
```
Comment (RetricSu) [inline docker/.build.mode.env:65]:
```suggestion
#	and output the logs in chain-reorg.log 

```

File: docker/web3/entrypoint.sh [hunk 3]
```diff
@@ -80,6 +80,7 @@ ETH_ACCOUNT_LOCK_HASH=$EthAccountLockCodeHash
 ROLLUP_TYPE_HASH=$RollupTypeHash
 PORT=8024
 CHAIN_ID=8521215115268
```
Comment (RetricSu) [inline docker/web3/entrypoint.sh:82]:
```suggestion
```

chain_id can be remove from env https://github.com/nervosnetwork/godwoken-web3/tree/compatibility-breaking-changes#config-database

File: docker/.build.mode.env (hunk 2 above)
Comment (RetricSu) [reply docker/.build.mode.env:65]: add new line at the end

**Pull request merged**